community/examples/quantum-circuit-simulator.yaml

```diff
@@ -0,0 +1,125 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+blueprint_name: quantum-circuit
+
+# Please review https://cloud.google.com/compute/docs/regions-zones
+# for availability of A2 machine types
+vars:
+  project_id:  ## Set project id here
+  deployment_name: qsim-demo
+  region: us-west4
+  zone: us-west4-b
+
+# Documentation for each of the modules used below can be found at
+# https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/modules/README.md
+
+deployment_groups:
+- group: primary
+  modules:
+  - source: modules/network/pre-existing-vpc
+    id: network1
+    kind: terraform
+  - source: modules/scripts/startup-script
+    id: quantum-simulator-setup
+    kind: terraform
+    settings:
+      runners:
+      - type: shell
+        destination: install-qsim.sh
+        content: |
+          #!/bin/bash
+          # This script implements https://quantumai.google/qsim/tutorials/gcp_gpu
+          set -e -o pipefail
+          curl -O https://raw.githubusercontent.com/GoogleCloudPlatform/compute-gpu-installation/main/linux/install_gpu_driver.py
+          python3 install_gpu_driver.py
+          curl -O https://repo.anaconda.com/miniconda/Miniconda3-py39_4.12.0-Linux-x86_64.sh
+          bash Miniconda3-py39_4.12.0-Linux-x86_64.sh -b -p /opt/conda
+          source /opt/conda/bin/activate base
+          conda init --system
+          # following channel ordering is important! use strict_priority!
+          # cuquantum comes from cuquantum label in nvidia channel
+          # libcutensor comes from main (default) label in nvidia channel
+          # cuda and all toolkit comes from cuda-11.5.2 label in nvidia channel
+          # everything else comes from conda-forge
+          conda config --system --set channel_priority strict
+          conda config --system --remove channels defaults
+          conda config --system --add channels conda-forge
+          conda config --system --add channels nvidia
+          conda config --system --add channels nvidia/label/cuda-11.5.2
+          conda config --system --add channels nvidia/label/cuquantum-22.07.1
+          conda update -n base conda --yes
+          conda create -n qsim python=3.9 --yes
+          conda install -n qsim cuda cuquantum make cmake cxx-compiler --yes
+          echo "cuda ==11.5.*" > /opt/conda/envs/qsim/conda-meta/pinned
+          conda clean -p -t --yes
+          conda activate qsim
+          pip install pybind11 cirq
+          git clone https://github.com/quantumlib/qsim.git /opt/qsim
+          cd /opt/qsim
+          export CUQUANTUM_DIR=/opt/conda/envs/qsim
+          make
+          pip install .
+      - type: data
+        destination: /var/tmp/qsim-example.py
+        content: |
+          import time
+          import cirq, cirq_google
+          import qsimcirq
+
+          # These defaults will take a long time on a single device!
```

Comment:
Defaults have been removed - this comment can be deleted.

Reply:
Done